> Ah, it seems like the lint step is failing (https://github.com/tigerbeetle/tigerbeetle/actions/runs/6275199313/job/17043780222?pr=1195). Can you run `zig fmt` on your changes (and you can run `zig build lint` locally) to confirm.

So, I had actually done that, but if I run `zig fmt src/tigerbeetle/repl.zig` then it fails one of your `zig build test` runs, with the below error.  Given that, I assumed the formatting was intentional so I tried to manually match the blocks of code.  I'm happy to tweak it however, just let me know what you'd like.  

```zig
run test: error: 'test.tidy' failed: tigerbeetle/repl.zig:792 error: line exceeds 100 columns
/home/ajr/workspace/tigerbeetle/src/tidy.zig:46:21: 0x45729c in test.tidy (test)
                    return error.LineTooLong;
```

--- a/src/tigerbeetle/repl.zig
+++ b/src/tigerbeetle/repl.zig
@@ -789,7 +789,10 @@ pub fn ReplType(comptime MessageBus: type) type {
                     );
 
                     if (lookup_account_results.len == 0) {
-                        try repl.fail("No such account or accounts exists.\n", .{});
+                        try repl.fail(
+                            "Failed to lookup account: {any}\n",
+                            .{LookupAccountResult.account_not_found},
+                        );
                     } else {
                         for (lookup_account_results) |*account| {
                             try repl.display_object(account);
@@ -818,7 +821,10 @@ pub fn ReplType(comptime MessageBus: type) type {
                     );
 
                     if (lookup_transfer_results.len == 0) {
-                        try repl.fail("No such transfer or transfers exists.\n", .{});
+                        try repl.fail(
+                            "Failed to lookup transfer: {any}\n",
+                            .{LookupTransferResult.transfer_not_found},
+                        );
                     } else {
                         for (lookup_transfer_results) |*transfer| {
                             try repl.display_object(transfer);
@@ -844,3 +850,23 @@ pub fn ReplType(comptime MessageBus: type) type {
         }
     };
 }
+
+pub const LookupAccountResult = enum(u32) {
+    ok = 0,
+    account_not_found = 1,
+    comptime {
+        for (std.enums.values(LookupAccountResult), 0..) |result, index| {
+            assert(@intFromEnum(result) == index);
+        }
+    }
+};
+
+pub const LookupTransferResult = enum(u32) {
+    ok = 0,
+    transfer_not_found = 1,
+    comptime {
+        for (std.enums.values(LookupTransferResult), 0..) |result, index| {
+            assert(@intFromEnum(result) == index);
+        }
+    }
+};

